lib/hammer_cli_csv/content_views.rb

```diff
@@ -9,6 +9,9 @@ class ContentViewsCommand < BaseCommand
       command_name 'content-views'
       desc         'import or export content-views'
 
+      option %w(--no-publish), :flag, _('Skip publishing content views on import')
+      option %w(--no-promote), :flag, _('Skip promoting content views on import')
```

Comment:
The validation is good to do, if possible. I would suggest that the options follow the hammer pattern of --[no-]publish
https://github.com/Katello/hammer-cli-csv/blob/master/lib/hammer_cli_csv/products.rb#L7